### PR TITLE
Add config-git descriptive interface reference 

### LINKED ISSUE #597: Snap package needs system-files plug for issue list command to work
**Description**

On Ubuntu 20.04, with glab 1.14.0-16-g222c0c5, the `glab issue list` command fails due to missing permissions.

It seems that `glab` is trying to access `/etc/gitconfig`. However, it cannot in a non `--classic` install (as recommended in the glab documentation), as the snap only has access to files in the user's home directory.

**Expected Behavior vs Actual Behavior**

- Expected: `glab issue list` works after successful authentication
- Actual: `glab issue list` fails with the following error:

> warning: unable to access '/etc/gitconfig': Permission denied

**Possible Fix**

1. Provide a [system-files](https://snapcraft.io/docs/system-files-interface) plug for the snap
2.  ~~Recommend to install with `sudo snap install glab --edge --classic`~~ => does not work either
3. Remove the snap and install the `.deb` package instead

**Steps to Reproduce**

```
$ glab issue list
warning: unable to access '/etc/gitconfig': Permission denied
warning: unable to access '/etc/gitconfig': Permission denied
fatal: unknown error occurred while reading the configuration files
git: exit status 128

$ sudo snap connect glab:system-files
error: snap "glab" has no plug named "system-files"
```

**Logs**

See above

**Your Environment**

* Version used (Run `glab --version`): 1.14.0-16-g222c0c5
* Operating System and version: Ubuntu 20.04

### PR DESCRIPTION
**Pull request by dplanella**
**Description**

Adds a [system-files interface](https://snapcraft.io/docs/system-files-interface) to the glab snap.

**Related Issue**

Resolves #597

**How Has This Been Tested?**

Not been tested yet, sending for early review.

**Screenshots (if appropriate):**

**Types of changes**

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation
- [x] Chore (Related to CI or Packaging to platforms)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -152,6 +152,7 @@ brew upgrade glab
 Make sure you have snap installed on your Linux Distro (https://snapcraft.io/docs/installing-snapd).
 1. `sudo snap install --edge glab`
 1. `sudo snap connect glab:ssh-keys` to grant ssh access
+1. `sudo snap connect glab:config-git` to grant read access to `/etc/gitconfig`
```
Comment (profclems) [inline README.md]: I would that we add a reference to the issue that led to this PR so users know why that access should be granted
```suggestion
1. `sudo snap connect glab:config-git` to grant read access to `/etc/gitconfig`. See https://github.com/profclems/glab/issues/597
```

**Review by profclems — COMMENTED** [review]
Thanks @dplanella for your contribution. Very much appreciated.

Just one suggestion... Aside that, everything looks good.

File: README.md (hunk 1 above)
Comment (dplanella) [reply README.md]: Sounds good, thanks!

**Comment by profclems** [comment]
@dplanella are you capable of testing if the snap works?

**Review by profclems — APPROVED** [review]

**Comment by renefritze** [comment]
If I'm still running into the issue of the glab snap not being able to access my `/etc/gitconfig` what's the current guidance to resolve that?

**Pull request merged**